tiledb/sm/c_api/tiledb.h

```diff
@@ -1058,6 +1058,38 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config);
  *    Ratio of the budget allocated for array data in the sparse global
  *    order reader. <br>
  *    **Default**: 0.1
+ * - `sm.mem.reader.sparse_global_order.ratio_result_tiles` <br>
```

Comment:
My biggest worry is that configuring all those parameters will be very cumbersome. We (TileDB) need to makes sure we find **excellent** default values before releasing this to the users.

Reply:
Yes the goal is to find defaults that most of our users will not have to change and to use those to debug customer's issues.